### DISCUSSION TITLE
请问一下软件多开问题

### DISCUSSION THREAD

**Discussion by Mio521** [mikumifa/biliTickerBuy#311] [提问专区]
如果要多开的话是要下多个软件然后在不同的文件夹里打开程序吗？还是可以下一个软件在同一文件夹里打开多个程序？谢谢

**Comment by mikumifa** [comment]
不同文件夹，目前相同文件夹没有实现这个功能

**Reply by Mio521** [reply]
谢谢大佬 再问一下一台设备可以登8个账号抢吗
